chip8_vm/src/instructions.rs

```diff
@@ -20,49 +20,60 @@ pub type Byte = u8;
 pub type Nibble = u8;
 
 
-/// Assembly opcode
+/// Raw instruction
+///
+/// Helper around the raw bits, not necessarily a valid instruction.
 #[derive(Copy)]
-pub struct Op {
-    raw: u16
+pub struct RawInstruction {
+    bits: u16
 }
 
-impl Op {
-    pub fn new(raw: u16) -> Op {
-        Op{ raw: raw }
+impl RawInstruction {
+    /// Creates a new raw instruction without any checks of the `bits`
+    pub fn new(bits: u16) -> RawInstruction {
+        RawInstruction{ bits: bits }
     }
 
+    /// The *raw bits*
     #[allow(dead_code)]
-    pub fn raw(&self) -> u16 {
-        self.raw
+    pub fn bits(&self) -> u16 {
+        self.bits
     }
 
+    /// The *address bits* part
     pub fn addr(&self) -> Addr {
-        self.raw & 0x0FFF
+        self.bits & 0x0FFF
     }
 
+    /// The *index of the `Vx` register* part
```

Comment:
I find this a tiny bit confusing, it sounds like this returns an index into bits. Maybe reword to: "The *`Vx` register-index* part"

Same for Vy